community/cloud-foundation/templates/ssl_certificate/examples/ssl_certificate.yaml

```diff
@@ -0,0 +1,59 @@
+# Example of the SSL certificate template usage.
+
+imports:
+  - path: templates/ssl_cerficate/ssl_cerficate.py
+    name: ssl_certificate.py
+
+resources:
+  - name: example-certificate
+    type: ssl_certificate.py
+    properties:
+      certificate: |
+        -----BEGIN CERTIFICATE-----
```

Comment:
Should be `<FIXME:certificateContents>`.

Reply:
Fixed.